index.html

```diff
@@ -0,0 +1,42 @@
+<!-- Behavior
+
+
+Put a border around the car information that matches the color of the car. To make this easier, set the color property value to an existing named CSS color.
+When you click on one of the car elements, change the width of the border to a higher value, and change the background color to any other color of your choosing.
+Also, on click of the car element, clear the value of the text input in the navbar, and put the cursor in the text input.
+When you start typing into the navbar's text input, the description of the currently selected car should be bound to what you are typing in and match it exactly.
+
+
+
+
+ -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Used Car Sales</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link rel = "stylesheet" href = "styles/main.css">
+</head>
+<body>
+  <!-- Put a standard Bootstrap navbar element at the top of your page. Put only two elements in the navbar.-->
+  <nav class = "navbar navbar-default">
+  <!-- A text input field. -->
+    <label for= "inputBox">Edit Description
+      <input type="text" name="textInput" id = "inputBox" placeholder="click on a card first" disabled = "true">
+    </label>
+  <!-- A button. -->
+    <button  id = "soldButton" class = "btn btn-default" disabled = "true">Set Sale Status</button> 
+  </nav>
+
+  <div id="output" class = "container-fluid">
+   <!-- car cards go here. -->
+  </div>
+<script src = "javascript/DOM.js"></script> <!--this had to go first in order for'outputDiv' variable to work. I don't like this, not truly modular. Change V2. -->
```

Comment:
It's fine, this DOM script doesn't even need to be an IIFE according to the directions.  If you'll notice in the example code, the quiz.js is just flat JavaScript doing stuff globally.